mlrun/runtimes/sparkjob/abstract.py

```diff
@@ -711,3 +752,40 @@ def _get_crd_info() -> Tuple[str, str, str]:
             AbstractSparkRuntime.version,
             AbstractSparkRuntime.plural,
         )
+
+    def _delete_resources(
```

Comment:
Note you also need to implement `_enrich_list_resources_response` I don't mind that it will be second phase, but it needs to be done